Tests/UnitTests/fss-search-results.component.spec.ts

```diff
@@ -175,11 +175,12 @@ describe('FssSearchResultsComponent', () => {
         let css = "[id='" + item.BatchID.value + "']";
         const home = fixture.debugElement.query(By.css(css)).nativeElement;
         if (item.allFilesZipSize) {
-          expect(home.querySelector("ukho-button").textContent).toEqual("Download all");
+          expect(home.querySelector("a").textContent).toEqual("Download all");
```

Comment:
Please add an expectation to confirm the 'Download all' link is enabled.
Please add an expectation to confirm the 'note' (ukho-dialogue) element is not visible

Reply:
Modified as per review comment